proto/osmosis/gamm/v1beta1/pool.proto

```diff
@@ -32,6 +35,57 @@ message PoolAsset {
   // ];
 }
 
+// Parameters for changing the weights in a balancer pool smoothly from
+// a start weight and end weight over a period of time.
+// Currently, the only smooth change supported is linear changing between
+// the two weights, but more types may be added in the future.
+// When these parameters are set, the weight w(t) for pool time `t` is the following:
+//   t <= start_time: w(t) = initial_pool_weights
+//   start_time < t <= start_time + duration: 
+//     w(t) = initial_pool_weights + (t - start_time) * 
+//       (target_pool_weights - initial_pool_weights) / (duration)
+//   t > start_time + duration: w(t) = target_pool_weights
+message SmoothWeightChangeParams {
+  // The start time for beginning the weight change.
+  // If a parameter change / pool instantiation leaves this blank,
```

Comment:
It should never be blank in the params, only in the msg creating the pool, right?

Reply:
Yes